### DISCUSSION TITLE
No wagmi config found. Ensure you have set up a config: https://wagmi.sh/react/config

### DISCUSSION THREAD

**Discussion by Lawrenxz** [wevm/wagmi#2675] [Q&A]
Hello! I am having a problem here in wagmi config , I just follow and copied the configuration of wagmi here [](https://docs.walletconnect.com/2.0/web/web3modal/react/wagmi/installation) then suddenly an error appear that the wagmi config is not fount. My working project is react 17 version, I tried it on react 18 and its working. Is that related to the problem? And can someone know how to fix this?

**Comment by tmm** [comment]
Please post a minimal example so folks can help you out.
